com.darhon.syncbackup.yml

```diff
@@ -0,0 +1,43 @@
+app-id: com.darhon.syncbackup
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: syncbackup
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --share=ipc
+  - --device=dri
+  - --share=network
+  - --filesystem=host
+modules:
+  - name: rsync
+    buildsystem: autotools
+    config-opts:
+      - --disable-zstd
+      - --disable-xxhash
+    sources:
+      - type: archive
+        url: http://archive.ubuntu.com/ubuntu/pool/main/r/rsync/rsync_3.2.3.orig.tar.gz
+        md5: 209f8326f5137d8817a6276d9577a2f1
+    cleanup:
+      - /share/man
+
+  - name: ksshaskpass
+    buildsystem: cmake
+    sources:
+      - type: archive
+        url: http://archive.ubuntu.com/ubuntu/pool/universe/k/ksshaskpass/ksshaskpass_5.22.5.orig.tar.xz
+        md5: 2d17696a462d4b1abc3f8b50811ba78c
+    cleanup:
+      - /share/man
+
+  - name: syncbackup
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+    sources:
+      - type: git
+        url: https://github.com/sigulete/flathub.git
```

Comment:
This repository doesn't contain the source code.

Also this REQUIRE a specific commit.

Reply:
Just to clarify. This shoud point to the code of the software you are packaging. Not the flathub package repository.